packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/nodejs/summaryofsteps.md

```diff
@@ -1 +1,50 @@
-## Summary of steps
+# Summary of steps
+
+The sequence of steps for self-service registration is described in the following three sequence diagrams:
+
+### Diagram 1: Start new user registration with password factor
+
+The following diagram illustrates the beginning of the registration process where the user initiates their sign-in and enters their password.
```

Comment:
try to avoid using sign in as a noun. so, we can either do:

```suggestion
The following diagram illustrates the beginning of the registration process where the user initiates the sign-in process and enters their password.
```
or

```suggestion
The following diagram illustrates the beginning of the registration process where the user signs in and enters their password.
```